src/libs/OptionsListUtils.js

```diff
@@ -353,10 +353,18 @@ function getAllReportErrors(report, reportActions) {
         if (TransactionUtils.hasMissingSmartscanFields(transaction)) {
             _.extend(reportActionErrors, {smartscan: ErrorUtils.getMicroSecondOnyxError('report.genericSmartscanFailureMessage')});
         }
+
+        if (TransactionUtils.isDistanceRequest(transaction) && TransactionUtils.hasRouteError(transaction)) {
+            _.extend(reportActionErrors, {waypoint: ErrorUtils.getLatestErrorField(transaction, 'route')});
+        }
     } else if ((ReportUtils.isIOUReport(report) || ReportUtils.isExpenseReport(report)) && report.ownerAccountID === currentUserAccountID) {
         if (ReportUtils.hasMissingSmartscanFields(report.reportID)) {
             _.extend(reportActionErrors, {smartscan: ErrorUtils.getMicroSecondOnyxError('report.genericSmartscanFailureMessage')});
         }
+
+        if (ReportUtils.hasRouteError(report.reportID)) {
```

Comment:
As I said last time, I don't think we ever display this error so we could get the error message directly from the route error.

I suppose this work's too, so NAB. However, we also need to set it where it's actually displayed.